Tests/src/artofillusion/object/SceneCameraTest.java

```diff
@@ -0,0 +1,504 @@
+/* Copyright (C) 2020 by Maksim Khramov
+
+   This program is free software; you can redistribute it and/or modify it under the
+   terms of the GNU General Public License as published by the Free Software
+   Foundation; either version 2 of the License, or (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful, but WITHOUT ANY 
+   WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A 
+   PARTICULAR PURPOSE.  See the GNU General Public License for more details. */
+
+
+package artofillusion.object;
+
+import artofillusion.Camera;
+import artofillusion.Scene;
+import artofillusion.animation.FilterParameterTrack;
+import artofillusion.animation.PoseTrack;
+import artofillusion.animation.WeightTrack;
+import artofillusion.image.ComplexImage;
+import artofillusion.image.filter.ImageFilter;
+import artofillusion.math.BoundingBox;
+import artofillusion.math.CoordinateSystem;
+import artofillusion.test.util.StreamUtil;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.IOException;
+import java.io.InvalidObjectException;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ *
+ * @author MaksK
+ */
+public class SceneCameraTest
+{
+  
+  @Test
+  public void testGetCameraBounds() {
+    SceneCamera camera = new SceneCamera();
+    BoundingBox cBox = camera.getBounds();
+    Assert.assertEquals(-0.25, cBox.minx, 0);
+    Assert.assertEquals(0.25, cBox.maxx, 0);
+    
+    Assert.assertEquals(-0.15, cBox.miny, 0);
+    Assert.assertEquals(0.20, cBox.maxy, 0);
+    
+    Assert.assertEquals(-0.20, cBox.minz, 0);
+    Assert.assertEquals(0.20, cBox.maxz, 0);
+  }
+  
+  
+  @Test
+  public void testCreateSceneCamera() {
+    
+    
+    SceneCamera camera = new SceneCamera();
+    ObjectInfo camObj = new ObjectInfo(camera, new CoordinateSystem(), "Camera");
+    Assert.assertTrue(camera.isPerspective());
```

Comment:
Which leaves this as the one assertion that is actually specific to a new `SceneCamera`

So. Is the fact that a new `SceneCamera` defaults to Perspective mode significant and important to test for?

If so, we should rename the test "newCameraPerspectiveByDefault()" or similar, to reflect what we are actually testing.

Reply:
Okay, better. I'm still not sure why it's important.